src/scenes/AcceleratorRouter/Scoring/PhaseScores/LastUpdated.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { useMemo } from 'react';
+
+import { Typography } from '@mui/material';
+
+import { useLocalization } from 'src/providers';
+import strings from 'src/strings';
+import { PhaseScores, Score } from 'src/types/Score';
+import { getLongDate } from 'src/utils/dateFormatter';
+
+interface LastUpdatedProps {
+  phaseScores: PhaseScores;
+}
+
+const LastUpdated = ({ phaseScores }: LastUpdatedProps) => {
+  const { activeLocale } = useLocalization();
+
+  const modifiedTime = useMemo(
+    () =>
+      phaseScores?.scores
+        .map((score: Score) => score.modifiedTime)
```

Comment:
Do we need to convert this to a numeric timestamp prior to sort?

Reply:
Nope, we get back full timestamps so they sort as expected as strings. I _do_ need to use `pop` instead of `shift` though... right now grabbing the earliest timestamp instead of latest.

Reply:
```
const dates = [
    "2024-03-20T15:21:46.208342Z",
    "2024-03-01T20:20:11.038378Z",
    "2024-03-19T20:20:11.038378Z",
    "2024-03-19T20:21:11.038378Z",
    "2024-03-19T20:20:11.038378Z",
    "2024-03-11T21:20:27.910470Z",
    "2024-03-19T20:20:11.038378Z",
    "2024-03-19T21:20:27.910470Z",
    "2024-03-19T20:20:11.038378Z",
    "2024-03-19T20:22:12.038378Z",
    "2024-03-21T20:21:11.038378Z",
    "2024-03-19T20:22:11.038378Z",
]

console.log(dates.sort())
```

If you want to check my work

Reply:
Thanks for the clarification. All good 🙌 , I realized we do the conversion elsewhere because of date yyyy-mm-dd formats instead of actual timestamps.